server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { AppVersion } from '../src/entities/app_version.entity';
+
+export class UpdateDefinitionsForGlobalSettings1639038616546 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const entityManager = queryRunner.manager;
+    const queryBuilder = queryRunner.connection.createQueryBuilder();
+    const appVersionRepository = entityManager.getRepository(AppVersion);
+
+    const appVersions = await appVersionRepository.find();
+
+    for (const version of appVersions) {
+      const definition = version['definition'];
+
+      if (!definition.globalSettings) {
```

Comment:
will throw error when definition is null

Reply:
ignore^

Reply:
Added empty check ^^